src/components/EventCalendar/EventCalendar.module.css

```
@@ -1,21 +1,38 @@
   margin-bottom: 20px;
-  background: rgb(255, 255, 255);
+  background: var(--grey-bg-color);
   border-radius: 10px;
   padding: 5px;
 }
+
 .calendar__header {
   display: flex;
   margin-bottom: 2rem;
   align-items: center;
   margin: 0px 10px 0px 10px;
```

Biome (1.9.4)

[error] Unexpected shorthand property margin after margin-bottom



(lint/suspicious/noShorthandPropertyOverrides)